src/components/DatasetItem.vue

```diff
@@ -7,6 +7,23 @@
       </dataset-info>
     </el-dialog>
 
+    <div class="opt-image">
+      <router-link :to="opticalImageAlignmentHref" v-if="opticalImageSmall && haveEditAccess && dataset.status != 'STARTED'" title="Edit Optical Image">
```

Comment:
I'd incapsulate these multiple checks into a computed property of the component. The same goes to `v-else-if` below